### ISSUE TITLE
duplicate `main` elements and IDs

### ISSUE THREAD

**Issue by iamjolly** [18F/uswds-jekyll#200]
Noticed when using the `page` layout... seems to be only this layout that has this problem.

The `page` layout specifies a `<main ... id="main-content" ... >` which is then included inside another `<main ... id="main-content" ... >` block within the `default` layout. 

Pages should have only one `main` element/region, and they should have no duplicate IDs. 

Suggest changing the `<main>` to a `<div>` in the `page` layout to fix.

**Comment by ryanwoldatwork** [comment]
@iamjolly, @ryanhofdotgov - Has this been addressed? Can this issue be closed now?

**Comment by ryanhofdotgov** [comment]
I don't think so? I've really only worked around it in my template, just following what folks did in ux-guide. The uswds-jekyll docs say "[post] is identical to the layout page", which isn't true anymore, and I don't quite understand the context of the original, cascading change:
https://github.com/18F/uswds-jekyll/pull/178/commits/7e663f095ee43289007e668e3688ccdfdbcff608#diff-2c19d9859b055d0302043d0fa2833e3f

In short, I'm not sure if this is a bug or a feature.

**Comment by ryanwoldatwork** [comment]
Thanks @ryanhofdotgov

**Comment by ryanwoldatwork** [comment]
https://github.com/18F/uswds-jekyll/pull/206

**Comment by mgwalker** [comment]
Closing as stale, on the assumption that the need will resurface if it still exists.
